packages/client/ts/lib/astar.ts

```diff
@@ -1,4 +1,4 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+/* eslint-disable */
```

Comment:
We could install all libraries from "lib" directory as npm dependencies.

Reply:
Clarify

Reply:
Ok, I see there is actually one library which can be an npm dependency - https://www.npmjs.com/package/astar-andrea. My proposal is to use `astar-andrea` as an external dep instead of keeping it in `lib` directory. Just a suggestion, no rush to make it happen in this PR.